seed/challenges/01-responsive-web-design/applied-visual-design.json

```diff
@@ -1195,7 +1196,7 @@
         "    <div class=\"cardText\">",
         "      <strong><h4>Alphabet</h4></strong>",
         "      <hr>",
-        "      <em><p>Google was founded by Larry Page and Sergey Brin while they were Ph.D. students at Stanford University.</p></em>",
+        "      <em><p>Google was founded by Larry Page and Sergey Brin while they were Ph.D. students at <strong>Stanford University</strong>.</p></em>",
```

Comment:
Every challenge from "Make the text bold" up to here needs this change, not just the ending and beginning one.